11-payment-encoding.md

```diff
@@ -153,7 +153,7 @@ Currently defined tagged fields are:
 ### Requirements
 
 A writer:
-  - MUST include exactly one `p` and `s` fields.
+  - MUST include exactly one `p`, `9` (with at least the `payment_secret` support bit set), and `s` fields.
```

Comment:
I believe this would be too specific.
What's probably confusing the requirement to include an `s` field without any mention of the feature bits.
Here is what I propose:

```suggestion
  - MUST include exactly one `p` field.
  - If the `payment_secret` feature is set, MUST include exactly one `s` field.
```

Reply:
Note that the requirements around features are in the `Feature Bits` section, so we could even remove that additional requirement which ends up being duplicated.

```suggestion
  - MUST include exactly one `p` field.
```